command/plugin.go

```diff
@@ -1,7 +1,3 @@
-//
-// This file is automatically generated by scripts/generate-plugins.go -- Do not edit!
-//
-
```

Comment:
Do you think it would be a good idea to keep that one being auto-generated, and have another 'plugins_manual.go' file ? It could for example add things through an init.

Reply:
I like this idea. Do you have any specific requirements for how the components should be stored or can I create similar maps and used them in plugins.go. Trying to understand what you are thinking so I can write the file with global components in mind.

Reply:
No requirement that I can think if; I'd probably create a similar map and then copy it over 🤔